arches_search/models/models.py

```diff
@@ -0,0 +1,180 @@
+        db_table = "arches_search_numeric"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["tileid", "resourceinstanceid", "node_alias"],
+                name="search_numeric_unique",
+            )
+        ]
+        indexes = [
+            models.Index(fields=["tileid", "node_alias", "value"]),
+            models.Index(fields=["tileid"]),
+            models.Index(fields=["resourceinstanceid"]),
+            models.Index(fields=["datatype"]),
+            models.Index(fields=["graph_alias"]),
+            models.Index(fields=["node_alias"]),
+            models.Index(fields=["value"]),
+        ]
+
+
+class UUIDSearch(models.Model):
+    id = models.AutoField(primary_key=True)
+    tileid = models.UUIDField()
+    resourceinstanceid = models.UUIDField()
+    graph_alias = models.TextField()
+    node_alias = models.TextField()
+    datatype = models.TextField()
+    value = models.UUIDField()
+
+    class Meta:
+        managed = True
+        db_table = "arches_search_uuid"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["tileid", "resourceinstanceid", "node_alias"],
+                name="search_uuid_unique",
+            )
+        ]
+        indexes = [
+            models.Index(fields=["tileid", "node_alias", "value"]),
+            models.Index(fields=["tileid"]),
+            models.Index(fields=["resourceinstanceid"]),
+            models.Index(fields=["datatype"]),
+            models.Index(fields=["graph_alias"]),
+            models.Index(fields=["node_alias"]),
+            models.Index(fields=["value"]),
+        ]
+
+
+class DateSearch(models.Model):
+    id = models.AutoField(primary_key=True)
+    tileid = models.UUIDField()
+    resourceinstanceid = models.UUIDField()
+    graph_alias = models.TextField()
+    node_alias = models.TextField()
+    datatype = models.TextField()
+    value = models.DateField()
+
+    class Meta:
+        managed = True
+        db_table = "arches_search_date"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["tileid", "resourceinstanceid", "node_alias"],
+                name="search_date_unique",
+            )
+        ]
+        indexes = [
+            models.Index(fields=["tileid", "node_alias", "value"]),
+            models.Index(fields=["tileid"]),
+            models.Index(fields=["resourceinstanceid"]),
+            models.Index(fields=["graph_alias"]),
+            models.Index(fields=["node_alias"]),
+            models.Index(fields=["datatype"]),
+            models.Index(fields=["value"]),
+        ]
+
+
+class DateRangeSearch(models.Model):
+    id = models.AutoField(primary_key=True)
+    tileid = models.UUIDField()
+    resourceinstanceid = models.UUIDField()
+    graph_alias = models.TextField()
+    node_alias = models.TextField()
+    datatype = models.TextField()
+    start_date = models.DateField()
+    end_date = models.DateField()
```

Comment:
Should be IntegerFields here.
Also, because we're using "value" everywhere else, maybe here we should use "start_value" and "end_value"?